### ISSUE TITLE
`Bigcommerce::getTime()` doesn't validate store credentials and returns garbage

### ISSUE THREAD

**Issue by luke8086** [bigcommerce/bigcommerce-api-php#298]
### Expected behavior

`\Bigcommerce\Api\Client::getTime()` should fail on invalid store credentials, as [documented](https://github.com/bigcommerce/bigcommerce-api-php/blob/master/src/Bigcommerce/Api/Client.php#L477)

### Actual behavior

[`\Bigcommerce\Api\Client::getTime()`](https://github.com/bigcommerce/bigcommerce-api-php/blob/master/src/Bigcommerce/Api/Client.php#L483) uses `$api_url` instead of `$api_path` so it effectively requests `https://api.bigcommerce.com/time`, which doesn't check for any store credentials.

On top of that, that endpoint returns timestamp in milliseconds, and `getTime()` parses is as seconds, so it returns bogus `DateTime` object

### Steps to reproduce behavior

Run without any prior setup:

```
>>> \Bigcommerce\Api\Client::getTime()
=>  DateTime @1718280499756 {#5814
      date: 56420-02-21 18:49:16.0 +00:00,
    }
```

**Comment by TomA-R** [comment]
Thank you for reporting this issue, @luke8086! I don't believe we have plans to add authentication to the /time endpoint but you're absolutely correct that `\Bigcommerce\Api\Client::getTime()` returns garbage. I've opened https://github.com/bigcommerce/bigcommerce-api-php/pull/299 to address this.

**Comment by luke8086** [comment]
Thank you @TomA-R for the quick response! 

I can see you already have an authenticated `/time` endpoint - https://developer.bigcommerce.com/docs/rest-management/store-information#get-system-timestamp

You just need to switch `getTime()` from `$api_url` to `$api_path` to use it  🙂 

```php
>>> $url = \Bigcommerce\Api\Client::$api_path . "/time"
=>  "https://api.bigcommerce.com/stores/6jvqbmuz8y/v2/time"

>>> $resp = \Bigcommerce\Api\Client::getConnection()->get($url)
=>  {#5817
      +"time": 1718286038,
    }

>>> new DateTime("@{$resp->time}")
=>  DateTime @1718286038 {#5799
      date: 2024-06-13 13:40:38.0 +00:00,
    }
```

**Comment by TomA-R** [comment]
> Thank you @TomA-R for the quick response!
> 
> I can see you already have an authenticated `/time` endpoint - https://developer.bigcommerce.com/docs/rest-management/store-information#get-system-timestamp
> 
> You just need to switch `getTime()` from `$api_url` to `$api_path` to use it 🙂
> 
> ```
> >>> $url = \Bigcommerce\Api\Client::$api_path . "/time"
> =>  "https://api.bigcommerce.com/stores/6jvqbmuz8y/v2/time"
> 
> >>> $resp = \Bigcommerce\Api\Client::getConnection()->get($url)
> =>  {#5817
>       +"time": 1718286038,
>     }
> 
> >>> new DateTime("@{$resp->time}")
> =>  DateTime @1718286038 {#5799
>       date: 2024-06-13 13:40:38.0 +00:00,
>     }
> ```

Thank you @luke8086, that's a valid point. Since that would change the behaviour and prerequisites of the call, I imagine that changing this might cause some frustration for people who are using this to simply check API connectivity. So perhaps introducing a `::getStoreTime()` method would make sense here, what do you think?

**Comment by luke8086** [comment]
@TomA-R  Yep, makes sense!  I agree about adding a new method to avoid breaking it for existing users.

Note though that both the [docstring](https://github.com/bigcommerce/bigcommerce-api-php/blob/master/src/Bigcommerce/Api/Client.php#L477) and the [README](https://github.com/bigcommerce/bigcommerce-api-php/tree/master?tab=readme-ov-file#connecting-to-the-store) say that `getTime()` tests connection to the store, not just the API, so they'll need to be updated.

**Comment by TomA-R** [comment]
Thanks again for reporting! The fix has been merged and released in version 3.3.7.
